api/build.gradle

```diff
@@ -51,6 +51,7 @@ signing {
 }
 
 dependencies {
+	compileOnly("com.google.errorprone:error_prone_annotations:${errorproneVersion}")
```

Comment:
What is this dependency needed for?

Reply:
Needed for `@CheckReturnValue`. As recommended [here](https://errorprone.info/bugpattern/CheckReturnValue#fnref:jsr)